pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -48,6 +48,14 @@ var dummyServiceInstance = &model.ServiceInstance{
 	},
 }
 
+var blackholeFilters = []*listener.Filter{{
```

Comment:
this will probably suffer from https://github.com/istio/istio/pull/29528 (envoyfilter applied will impact all proxies). but we may want to fix it in another way

Reply:
Yeah,  Not sure if people use envoy filters for blackhole filter/cluster. IMO we should let this in to avoid frequent Any conversions but if you think it would be useful - I will put it back